package.json

```diff
@@ -13,7 +13,8 @@
         "test:unit:coverage": "sfdx-lwc-jest --coverage",
         "prettier": "prettier --write \"**/*.{cls,cmp,component,css,html,js,json,md,page,trigger,xml,yaml,yml}\"",
         "prettier:verify": "prettier --list-different \"**/*.{cls,cmp,component,css,html,js,json,md,page,trigger,xml,yaml,yml}\"",
-        "postinstall": "node bin/check-version.js"
+        "postinstall": "node bin/check-version.js && husky install",
+        "precommit": "npm run prettier && npm run test"
```

Comment:
what about lint?

Reply:
@albarivas if you look a couple of lines up on line 9, lint is included in `npm run test` (I know, it's weird, not my choice)